cypress/commons/constants/generic/TestConstants.js

```diff
@@ -159,5 +159,5 @@ export const SCENARIO_RUN_IN_PROGRESS = 'Scenario run in progress...';
 
 export const WEBAPP_URL_REGEX = {
   SCENARIO_PAGE: new RegExp(PAGE_NAME.SCENARIO + '$'),
-  SCENARIO_PAGE_WITH_ID: new RegExp(PAGE_NAME.SCENARIO + '/(s-[\\w]+)'),
+  PAGE_WITH_SCENARIOID: /\/(s-[\w]+)/,
```

Comment:
_cypress/integration/brewery/CreateScenario.spec.js_ uses another constant : `URL_REGEX.SCENARIO_PAGE_WITH_ID` ([line 20](https://github.com/Cosmo-Tech/azure-sample-webapp/blob/main/cypress/commons/constants/generic/TestConstants.js#L23)), `WEBAPP_URL_REGEX.SCENARIO_PAGE_WITH_ID` was used only in _routeUtils.js_ :slightly_smiling_face: 
This regex tests if any url ends with _/s-something_ because we have now two views with `scenarioId` in url, would it be better to call it `URL_WITH_SCENARIOID` or `IS_SCENARIOID_IN_URL` ?

Reply:
My bad, you're right, I didn't catch the `WEBAPP_URL_REGEX` vs. `URL_REGEX` difference :sweat_smile: 

Regarding the other constant name, maybe we could use something generic like `SCENARIO_ID_PATTERN`